roles/hosted_engine_setup/tasks/bootstrap_local_vm/03_engine_initial_tasks.yml

```diff
@@ -53,6 +53,10 @@
     with_items:
       - "OVESETUP_DWH_DB/password=str:{{ he_dwh_db_password }}"
     when: he_dwh_db_password is defined
+  - name: Setup Repositories on Local VM
+    import_role:
+      name: ovirt.ovirt.repositories
```

Comment:
Please rename the role to:
`@NAMESPACE@.@NAME@.repositories`

It will be replaced by CI depending on which version we will build